### DISCUSSION TITLE
Rebranding guide..

### DISCUSSION THREAD

**Discussion by pkking** [fedora-copr/copr#2451] [Q&A] [answered]
Hi, we [openEuler][https://www.openeuler.org/) are setting up our copr instance as `openEuler copr`, but im wondering if the rebranding is permitted and which parts can be rebranded  :)
Things comes to my mind were:
- GPG key domain name
- the name/logo of the instance(mostly in web templates)
- `vendor` field in rpm metadata

Some of the modifies will be code changes, so i'd like make them configurable amd create some PRs.

Appreciated any of the voice :) @xsuchy

**Comment by FrostyX** [comment, marked as answer]
Hello @pkking,

> but im wondering if the rebranding is permitted and which parts can be rebranded :)

the `copr-frontend` package is licensed under GPL-2.0-or-later so you can rebrand anything you want as long as you abide by the license. 

> the name/logo of the instance(mostly in web templates)

Apart from the Fedora Copr instance, we are also running an internal RedHat Copr instance. It has a custom logo and color scheme, so I guess the same kind of changes that you are going to make. The sources are available here https://pagure.io/rhcopr/design , hopefully, they will help you a bit.

> Some of the modifies will be code changes, so i'd like make them configurable amd create some PRs.

Sounds good to me

**Comment by xsuchy** [comment]
I second to @FrostyX , we already did some work, which allows customization. If you need more, then we will happily accept any pull request.
BTW if you want to discuss stuff before coding it, you are more than welcome to join https://lists.fedoraproject.org/archives/list/copr-devel@lists.fedorahosted.org/
